components/dropdown/README.md

```diff
@@ -21,190 +29,230 @@
 
 If the dropdown is initially empty when it's opened, the dropdown pointer will not be positioned correctly.  In such cases, the `no-auto-open` attribute may be added to the opener, enabling you to take control of when the dropdown is actually opened:
```

Comment:
This sentence becomes a bit strange without the code below

Reply:
I changed the `:`  to a `.`, I think it makes sense to keep this line in here. Do you think it would be useful to add a code example back?

Reply:
I think so, though I think you could combine the two code chunks, e.g.:
```html
<script>

dropdown.addEventListener('click', function() {
  // fetch some content
  ...
  // take control of when the dropdown is actually opened
  dropdown.toggleOpen();
</script>
<d2l-dropdown no-auto-open>
  ...
</d2l-dropdown>
```